simbev/helpers/helpers.py

```diff
@@ -56,13 +83,154 @@ def export_metadata(
+
+    start_date = start_date.date()
+    number_of_days = end_date-start_date
+    number_of_days = number_of_days.days + 1
+
+    # general
+    analysis_mid_dict["average_trip_count"] = round(df["drive_count"].mean()/number_of_days, 4)
+    analysis_mid_dict["average_drive_time"] = round(df["average_driving_time"].mean(), 4)
+    analysis_mid_dict["average_distance"] = round(df["average_distance"].mean(), 4)
+
+    analysis_mid_dict["charging_event_share_private"] = round(df[["private_count"]].sum().iloc[0] /
+                                                              (df[["private_count"]].sum().iloc[0] +
+                                                               df[["public_count"]].sum()).iloc[0], 4)
+    analysis_mid_dict["charging_event_share_public"] = round(df[["public_count"]].sum().iloc[0] /
+                                                             (df[["private_count"]].sum().iloc[0] +
+                                                              df[["public_count"]].sum()).iloc[0], 4)
+
+    # by car-type
+    # trip count by day
+    for vehicle in vehicle_array:
+        analysis_mid_dict["by_car_type"]["average_trip_count"][vehicle] = round(df["drive_count"].loc[
+                                                                                   df["car_type"] == vehicle
+                                                                               ].mean()/number_of_days, 4)
+    # average drive time by trip
+    for vehicle in vehicle_array:
+        analysis_mid_dict["by_car_type"]["average_drive_time"][vehicle] = round(df["average_driving_time"].loc[
+                                                                                   df["car_type"] == vehicle
+                                                                               ].mean(), 4)
+    # average distance by trip
+    for vehicle in vehicle_array:
+        analysis_mid_dict["by_car_type"]["average_distance"][vehicle] = round(df["average_distance"].loc[
+                                                                                   df["car_type"] == vehicle
+                                                                               ].mean(), 4)
+    # by destination
+    for destination in destination_array:
+        analysis_mid_dict["by_destination"]["average_distance"][destination] = round(df[destination].loc[
+                                                                                         df[destination]
+                                                                                         != (-1)].mean(), 4)
+    # save json-file
+    with open(Path(directory, "analysis_mid_{}.json".format(region_id)), "w") as outfile:
+        json.dump(analysis_mid_dict, outfile, indent=4, sort_keys=False)
+
+
+def timeitlog(timing):
+    def decorator(func):
+        os_path = Path(__file__).parent.parent
+        path_to_log_file = Path(os_path, 'results', 'log_file_simbev.txt')
```

Comment:
I get this error when running:
`FileNotFoundError: [Errno 2] No such file or directory: '/home/moritz/github/simbev/simbev/results/log_file_simbev.txt'`

Probably related to the new structure and missing results directory

Reply:
That must be the reason I am solving this problem.